doc/source/data/examples/batch_inference_object_detection.ipynb

```diff
@@ -0,0 +1,350 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Batch Image Object Detection using Ray Data\n",
+    "\n",
+    "In this example, we will demostrate how to use Ray Data to do batch object detection with a PyTorch model on a large set of images.\n",
+    "\n",
```

Comment:
What needs to be installed to run this example? See https://docs.ray.io/en/latest/ray-air/examples/pytorch_resnet_batch_prediction.html

Reply:
Also let's specify what instance type this was run on.